pkg/console/operator/sync_v400.go

```diff
@@ -106,12 +108,21 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 	// but we will handle the state of the operand below
 	co.ConditionResourceSyncSuccess(operatorConfig)
 	// the operand is in a transitional state if any of the above resources changed
-	// or if we have not settled on the desired number of replicas
-	if toUpdate || actualDeployment.Status.ReadyReplicas != deploymentsub.ConsoleReplicas {
+	// or if we have not settled on the desired number of replicas or deployment is not uptodate.
+	if toUpdate {
```

Comment:
Here I'm setting and unsetting the Progressing Status.
If any of the resources is syncing set the `ConditionResourceSyncProgressing`, else if requirements  for updating the version are not fulfilled, set the `ConditionResourceSyncProgressing` with the message mentioned in Jessica's mail. Otherwise just set the version.